src/ddmd/mars.d

```diff
@@ -245,6 +245,29 @@ extern (C++) void genCmain(Scope* sc)
     rootHasMain = sc._module;
 }
 
+private void semantic3OnDependencies(Module m, StringTable *importArray)
+{
+    if(!m)
+        return;
+
+    const(char) *mName = m.toChars();
+    size_t mNameLen    = strlen(mName);
+
+    if (importArray.lookup(mName, mNameLen))
+        return;
+
+    if (global.params.verbose)
+        fprintf(global.stdmsg, "semantic3 %s\n", mName);
+
+    importArray.insert(mName, mNameLen, cast(void*) mName);
```

Comment:
If we had a time machine, yes.  Re-formatting dmd's source at this point would not add value.

The DMD rules are based off what the 'prevailing' D style was at the time.  Since the style guide didn't specify for space after `cast` I counted the number of each type in phobos, which was fairly evenly split.  So I picked the one that closest matched the existing C++ style.

Since then Andrei decided/declared that a space should be added in phobos code, but IMO that has very little bearing on what style should be used in DMD.